db/models.js

```diff
@@ -23,3 +23,24 @@ exports.fetchAllArticles = () => {
       return arrayOfArticles;
     });
 };
+
+exports.selectArticlesById = (article_id) => {
+  return db
+    .query("SELECT * FROM articles WHERE article_id = $1;", [article_id])
+    .then(({ rows }) => {
+      const article = rows[0];
+      if (!article) {
+        return Promise.reject({ status: 404, msg: "Article not found" });
+      } else {
+        return article;
+      }
+    });
+};
+
+exports.selectCommentsByArticleId = (article_id) => {
+  return db
+    .query("SELECT * FROM comments WHERE article_id = $1;", [article_id])
+    .then(({ rows }) => {
```

Comment:
Will need to add more functionality to this function to pass the tests I listed above 🧪